snappy/DiskCache_Config.py

```diff
@@ -1,5 +1,6 @@
 import os
 import json
+from configUpdate import configUpdate
```

Comment:
If you're going to do a relative import, then this should be:

```
from .configUpdate import configUpdate
```

However, I don't think these files will ever exist outside of snappy, so I'd do this instead:

```
from snappy.configUpdate import ConfigUpdate
```